tests/sandbox/create_message_batches/test_field_validation.py

```diff
@@ -67,7 +70,19 @@ def test_invalid_body(nhsd_apim_proxy_url, correlation_id):
 @pytest.mark.parametrize("correlation_id", CORRELATION_IDS)
 def test_property_missing(nhsd_apim_proxy_url, property, pointer, correlation_id):
     """
-    .. py:function:: Test missing properties
+    .. py:function:: Scenario: A user submitting a request without a required attribute \
```

Comment:
I think we need to add a list of the properties we're checking for here

Reply:
Can be a partial with the values in as we use the same list on other tests

Reply:
Also this _missing_properties value should actually be the value coming from the constants - check the other envs tests!

Reply:
Addressed in request_properties.rst, linked below given/when/then